test/coverlet.core.tests/Extensions/StringExtensionsTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using Xunit;
+
+namespace Coverlet.Core.Tests.Extensions
+{
+    [ExcludeFromCodeCoverage]
+    public class StringExtensionsTest
```

Comment:
This should no longer be needed

Reply:
Yes, I forgot to remove it. I will fix it...

Reply:
I remebered - I'm added this attribute  to exclude lines from dotCover cove coverage analysis when I'm run all tests.

Reply:
I meant that since you'll be removing the `StringExtensions` class there should be no need for this test